Multisig/Logic/PrivateKeyController.swift

```diff
@@ -123,8 +158,8 @@ class PrivateKeyController {
                 try PrivateKey.deleteAll()
             }
 
-            // delete all key infos whos private keys do not exist
-            let infos = try KeyInfo.all().filter { !$0.hasPrivateKey }
+            // delete all device key infos whos private keys do not exist
+            let infos = try KeyInfo.all().filter { $0.keyType == .device && !$0.hasPrivateKey }
             try infos.forEach { try $0.delete() }
```

Comment:
do we need clean up for the wallet connect keys as well?

Reply:
no, only for device keys